kite-api/src/resolvers/user.resolver.js

```diff
@@ -20,31 +27,35 @@ const UserResolvers = {
     }
   },
   loginUser: async ({ ...userData }, req) => {
-    try {
-      const userDB = await loginUser(userData, req);
+    const userDB = await loginUser(userData, req);
```

Comment:
The nomenclature of this variable feels misleading, at first glance one might assume it to be something that represents an interface with a database called **users**. I know this is not true, however, so it does not need to be changed. Just a note. Perhaps "user" or "userObject" could take it's place, should you make that call

Reply:
In this context you are right, it could just be called user. The reason I named it userDB is because inside the loginUser() function I have 3 instances of a user, 1 from the database, 1 in Redis store, and 1 from the front-end, and I differentiate between them by appending where they are from to the end of "user" - Ex. userDB from database and userRedis from Redis.